README.md

```diff
@@ -1 +1,34 @@
-# leap-stats
\ No newline at end of file
+# leap-stats
+
+### Setup
+
+- Clone the repo
+```
+$ git clone git@github.com:leapdao/leap-stats.git
+```
+
+- Install dependencies
+
+```
+$ cd leap-stats
+$ yarn install
+```
+
+- Setup [`serverless`](https://serverless.com/framework/docs/getting-started/) local environment
+
+- Setup offline DynamoDB
+```
+$ serverless dynamodb install
+```
+
+- Run the local server
+```
+$ serverless offline start
+```
+
+### Deployment
+- Configure AWS Lambda account using [`aws-cli`](https://serverless.com/framework/docs/providers/aws/guide/credentials#setup-with-the-aws-cli) as per the instructions in the link
```

Comment:
Probably will be better to put link under the word link, instead of aws-cli - me, for example, had hard time to noticing where it is ^__^

Reply:
Sure, I'll update that 😅